src/main/java/lotto/domain/LottoPlay.java

```diff
@@ -0,0 +1,41 @@
+package lotto.domain;
+
+import java.util.List;
+import lotto.view.InputView;
+
+public class LottoPlay {
+
+    private final LottoTickets lottoTickets;
+    private final LottoMachine lottoMachine;
+    private final Analyzer analyzer;
+
+    public LottoPlay(InputView inputView) {
+        final int totalPrice = inputView.getTotalPrice();
+        this.lottoTickets = new LottoTickets(totalPrice);
+        this.lottoMachine = new LottoMachine(inputView.getWinNumbers(), inputView.getBonusNumber());
+        this.analyzer = new Analyzer(totalPrice);
+    }
+
+    public void run() {
+        analyzer.calculateTotalWinningMoney(lottoTickets.countCorrectWinNumber(lottoMachine),
```

Comment:
하나의 라인에 모든 작업을 처리하려고 하면 가독성이 떨어질 수 있어요!
여러개의 라인으로 분리해보면 어떨까요?
```suggestion
        final List<Integer> correctWinNumber = lottoTickets.countCorrectWinNumber(lottoMachine);
        final List<Integer> hasBonusNumber = lottoTickets.checkBonusNumber(lottoMachine);
        analyzer.calculateTotalWinningMoney(correctWinNumber, hasBonusNumber);
```

Reply:
여기도 `LottoTickets`에 있는 `Lotto`별로 정답과 일치하는 개수와,  bonus일치여부를 상태로 가지고 있는 별도의 객체를 만들어보면 어떨까요?

Reply:
WinningResult 라는 객체를 통해 관리하도록 수정했습니다. 😊